joblib/parallel.py

```diff
@@ -414,10 +414,10 @@ class Parallel(Logger):
         >>> from time import sleep
         >>> from joblib import Parallel, delayed
         >>> r = Parallel(n_jobs=2, verbose=5)(delayed(sleep)(.1) for _ in range(10)) #doctest: +SKIP
-        [Parallel(n_jobs=2)]: Done   1 out of  10 | elapsed:    0.1s remaining:    0.9s
-        [Parallel(n_jobs=2)]: Done   3 out of  10 | elapsed:    0.2s remaining:    0.5s
-        [Parallel(n_jobs=2)]: Done   6 out of  10 | elapsed:    0.3s remaining:    0.2s
-        [Parallel(n_jobs=2)]: Done   9 out of  10 | elapsed:    0.5s remaining:    0.1s
+        [Parallel(n_jobs=2)]: Done   1 tasks      | elapsed:    0.1s
+        [Parallel(n_jobs=2)]: Done   3 tasks      | elapsed:    0.2s
+        [Parallel(n_jobs=2)]: Done   6 tasks      | elapsed:    0.3s
+        [Parallel(n_jobs=2)]: Done   9 tasks      | elapsed:    0.5s
```

Comment:
Is that the ouptut you actually get when running the snippet above?

```py
from time import sleep
from joblib import Parallel, delayed
r = Parallel(n_jobs=2, verbose=5)([delayed(sleep)(.1) for _ in range(10)]) #doctest: +SKIP
```

Output:
```
[Parallel(n_jobs=2)]: Done  10 out of  10 | elapsed:    0.7s remaining:    0.0s
[Parallel(n_jobs=2)]: Done  10 out of  10 | elapsed:    0.7s finished
```

This may be because of short task batching. I think the easiest thing to do is to use a snippet like this (longer sleep time prevents task batching):
```py
from time import sleep
from joblib import Parallel, delayed
r = Parallel(n_jobs=2, verbose=10)([delayed(sleep)(.2) for _ in range(10)]) #doctest: +SKIP
```

On my machine the output is:
```
[Parallel(n_jobs=2)]: Done   1 tasks      | elapsed:    0.4s
[Parallel(n_jobs=2)]: Done   4 tasks      | elapsed:    0.6s
[Parallel(n_jobs=2)]: Done  10 out of  10 | elapsed:    1.2s finished
```

Reply:
Yeah, I did get fewer messages than were printed there; I assumed this varied from computer to computer, so I stuck to the structure of the original output. I'll replace it with my actual computer output to make sure it's realistic.